backend/src/main/kotlin/com/hiper2d/repository/RoomRepository.kt

```diff
@@ -1,8 +1,8 @@
 package com.hiper2d.repository
 
-import com.hiper2d.model.Echo
+import com.hiper2d.model.Room
 import org.springframework.data.mongodb.repository.ReactiveMongoRepository
 import org.springframework.stereotype.Repository
 
 @Repository
-interface EchoRepository: ReactiveMongoRepository<Echo, String>
+interface RoomRepository: ReactiveMongoRepository<Room, String>, RoomRepositoryCustom
```

Comment:
Can't we just add methods from custom into this interface?

Reply:
No. This is the way how to Spring Data requires to write code when you need some implementation of the repository interface method. Check out the [doc](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.custom-implementations)